docs/en/setup/backend/backend-upgrade.md

```diff
@@ -0,0 +1,26 @@
+# Backend upgrade
+First and most important thing is, SkyWalking agent caching the key-value of the metadata, i.e., the service id and service name, the serice instance id and service instance name, the endpoint id and endpoint name. The backend will not working after you clear the metadata from elasticsearch database but not restart the agent.
+
+**Remember to convert metadata from the old models to the new models.** 
+
+There are three ways for upgrade SkyWalking.
+1. Upgrade without transform metadata
+1. Upgrade by transform metadata
+
+## Upgrade without transform metadata
+1. Shutdown the backend and clean the buffer data
+1. Shutdown the database and clean the models
```

Comment:
In this way, look like you are doing a gray upgrate, right? 

Then there is a tricky question here, is shutdown,clear backend and database are necessary steps?